#! /bin/sh /usr/share/dpatch/dpatch-run
## 05_schema_fixup.dpatch by Stephen Gran <sgran@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Correct primary key declaration

@DPATCH@
--- a/conf/dbschema.sql	(revision 89)
+++ b/conf/dbschema.sql	(working copy)
@@ -19,7 +19,7 @@
 	"gecos" character varying(128),
 	"homedir" character varying(256) NOT NULL,
 	"shell" character varying DEFAULT '/bin/bash' NOT NULL,
-	PRIMARY KEY ("username")
+	PRIMARY KEY ("uid")
 );
 
 CREATE TABLE "usergroups" (
